frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/redux/stateUpdate/vmSettings/vm-settings-tab-state-update.ts

```diff
@@ -22,6 +23,45 @@ import { windowsToolsStorage } from '../../initial-state/storage-tab-initial-sta
 import { getStorages } from '../../../selectors/selectors';
 import { prefillVmTemplateUpdater } from './prefill-vm-template-state-update';
 
+export const selectUserTemplateOnLoadedUpdater = (options: UpdateOptions) => {
+  const { id, dispatch, getState } = options;
+  const state = getState();
+  if (options.changedCommonData.has(VMWizardProps.userTemplates)) {
```

Comment:
how would this even work? 

```
if user templates changed
    do not do anything 
```

please look at the other updaters for examples on how these guards should work